test/common.sh

```diff
@@ -13,78 +13,79 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 set -eu -o pipefail
 set -x
 
-# see: https://github.com/stedolan/jq/issues/105
-# and: https://github.com/stedolan/jq/wiki/FAQ#general-questions
-function jqi() {
-	filename="${1}"
-	jqexpr="${2}"
-	jq "${jqexpr}" "${filename}" > "${filename}.tmp" && mv "${filename}.tmp" "${filename}"
-}
+ROOT="${DIR}/.."
+
+# see: https://github.com/stedolan/jq/issues/105 & https://github.com/stedolan/jq/wiki/FAQ#general-questions
+function jqi() { filename="${1}"; jqexpr="${2}"; jq "${jqexpr}" "${filename}" > "${filename}.tmp" && mv "${filename}.tmp" "${filename}" }
 
 function deploy() {
-        cluster_type=${ORCHESTRATOR:-"kubernetes"}
 	# Check pre-requisites
 	[[ ! -z "${INSTANCE_NAME:-}" ]] || (echo "Must specify INSTANCE_NAME" && exit -1)
 	[[ ! -z "${LOCATION:-}" ]] || (echo "Must specify LOCATION" && exit -1)
 	[[ ! -z "${CLUSTER_DEFINITION:-}" ]] || (echo "Must specify CLUSTER_DEFINITION" && exit -1)
-        which az || (echo "az must be on PATH" && exit -1)
-
-	if [[ "${cluster_type}" == "kubernetes" ]]; then
-            [[ ! -z "${SERVICE_PRINCIPAL_CLIENT_ID:-}" ]] || (echo "Must specify SERVICE_PRINCIPAL_CLIENT_ID" && exit -1)
-	    [[ ! -z "${SERVICE_PRINCIPAL_CLIENT_SECRET:-}" ]] || (echo "Must specify SERVICE_PRINCIPAL_CLIENT_SECRET" && exit -1)
-	    which kubectl || (echo "kubectl must be on PATH" && exit -1)
-	fi
+	[[ ! -z "${SUBSCRIPTION_ID:-}" ]] || (echo "Must specify SUBSCRIPTION_ID" && exit -1)
```

Comment:
turn off logging (set +x), so that secrets and other mbi data like sub id is not printed to traces.  And keep that as a practice for other secret and mbi like data throughout the scripts.